docs/index.md

```diff
@@ -108,16 +232,7 @@ const resultPerMonth = new Query(crimeData).filterByCategory(category).groupByYe
 
 display(lineChart(resultPerMonth, "date", "total"));
 ```
-## Rankings
-```js
-const years = getYears().map(String);
-const year = view(Inputs.select(years, {value: "2018", label: "Selecteer jaar:"}));
-const category_parallel = view(Inputs.select(categories, {value: "Graffiti", label: "Selecteer categorie:"}));
-```
 
-```js
-display(parallel(crimeData, year, category_parallel));
-```
 ## Amount of crimes
```

Comment:
Maybe we can already remove all these old bar charts? They are slow, clunky and if necessary we can quickly add better ones using observable plot. To start restructuring, I would remove them.

Reply:
I would replace them once the replacements are done. This way we have an example.